packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -2493,16 +2493,15 @@ class Navigator extends StatefulWidget {
   /// _does_ animate the new route, and delays removing the old route until the
   /// new route has finished animating.
   ///
-  /// The removed route is removed without being completed, so this method does
+  /// The removed route is removed and completed with null, so this method does
   /// not take a return value argument.
```

Comment:
tiny wording suggestion:

```suggestion
  /// The removed route is removed and completed with a `null` value.
```